keps/sig-autoscaling/2021-scale-from-zero/README.md

```diff
@@ -490,8 +556,22 @@ well as the [existing list] of feature gates.
 -->
 
 - [x] Feature gate (also fill in values in `kep.yaml`)
-  - Feature gate name: HPAConfigurableTolerance
-  - Components depending on the feature gate: `kube-controller-manager`
+  - Feature gate name: `HPAScaleToZero`
+  - Components depending on the feature gate: `kube-apiserver`
```

Comment:
where does the hpa controller live? kcm?

Reply:
The HPA controller runs inside the KCM. The flag is only needed in the API server, as the HPA already supports scaling down to zero, but only `minReplicas > 0` is allowed by the API server without this flag. The flag itself is already implemented since 1.16